tests/test_endpoint.py

```diff
@@ -23,7 +23,7 @@ def test_server_health_returns_200():
 def test_server_list_actions_returns_200():
     request, response = app.test_client.get("/actions")
     assert response.status == 200
-    assert len(response.json) == 3
+    assert len(response.json) == 4
```

Comment:
We'll have to update this for every subclass of `Action` we create while writing new tests.

Reply:
Let's add more detail here about all the action subclasses in the response.

Reply:
more comments or more assert statements?

Reply:
assertions if possible.